streamlit-python/local_test_env.yml

```diff
@@ -4,9 +4,10 @@ channels:
   - snowflake
 dependencies:
   - python=3.8
-  - snowflake-snowpark-python
+  - snowflake-snowpark-python>=1.15.0
   - pip
   - pip:
       - pytest
+      - snowflake-cli-labs>=2.0.0
```

Comment:
Why is this required?

Reply:
if you are in testing mode, you might want to use Snow CLI to deploy your app. It's annoying to have to deactivate and reactivate your conda environment to use CLI, or to test locally.

Reply:
I can remove the version requirement though to avoid confusion